### ISSUE TITLE
YouTube application

### ISSUE THREAD

**Issue by wimdenherder** [wong2/chat-gpt-google-extension#140]
Dear Wong2, your extension is really really nice. I wrote a YouTube application. Are you interested in having this in this extension? Or maybe you would prefer to have a seperate extension for YouTube? It summarizes video's based on their subtitles. It would be nice to develop this further: adding the possibility to ask questions about the youtube video. You can send multiple inputs to GPT and then ask 'could you summarize my last five inputs?' to handle big video's as well.

**Comment by wong2** [comment]
Hi, do you have a link of your extension?

**Comment by wimdenherder** [comment]
Here they are:
- I made a pull request: https://github.com/wong2/chat-gpt-google-extension/pull/142/commits/5e9d6ff10892a1625798045c5b4afb3832c4391a
- direct link to branch youtube on forked repository: https://github.com/wimdenherder/chat-gpt-google-extension/tree/youtube
Looking forward to hearing from you. Nice to collaborate on this!

**Comment by wong2** [comment]
I've seen a extension do this: https://www.producthunt.com/posts/youtube-summarizer-with-chatgpt

**Comment by wimdenherder** [comment]
I checked, but that extension:
- is not working yet
- in their promotional video, they jump to a new tab
- the way the summary result is included in your extension is way nicer

**Comment by wong2** [comment]
I see. I'm not planning to add this feature to this extension, as I believe a product should focus on one use case first. 
So I suggest you create another extension and compete with YouTube Summarizer with ChatGPT.

**Comment by josStorer** [comment]
I implement this feature in my repository https://github.com/josStorer/chatGPTBox

**Comment by wimdenherder** [comment]
> I implement this feature in my repository https://github.com/josStorer/chatGPTBox

Thank you, josStorer, it's a pleasure to be part of your open source project, great extension by the way, I like that you automatically show icons when selecting text (I was working on something similar)

Could you mention my github: https://github.com/wimdenherder on your page for the YouTube summary feature (or inspiration/credit), thanks a lot!

**Comment by josStorer** [comment]
Of course, I added it to the Features list, and I think this position is more noticeable.

![image](https://user-images.githubusercontent.com/13366013/226159146-6e6e827a-09ad-44d1-862b-a5fb7d05bf7f.png)

**Comment by wimdenherder** [comment]
Thank you, looks cool 😎 I've shared your extension on my linkedin: https://www.linkedin.com/feed/update/urn:li:share:7043116459245367296/
